server/miljø.ts

```diff
@@ -0,0 +1,38 @@
+import logger from './logger';
+
+const lokaltMiljø = {
+  dokumentUrl: 'http://localhost:8082',
+  apiUrl: 'http://localhost:8091',
+  tokenxRedirectUri:
+    'https://localhost:8080/familie/alene-med-barn/soknad/oauth2/callback',
+};
+
+const devMiljø = {
+  dokumentUrl: 'https://familie.dev.nav.no/familie/dokument',
+  apiUrl: 'https://familie.dev.nav.no/familie/alene-med-barn/soknad-api',
```

Comment:
Ettersom vi kaller soknad-api direkte nå fra node-backend så burde vi kunne bruke service-discover url's her
```suggestion
  dokumentUrl: 'http://familie-dokument,
  apiUrl: 'http://familie-soknad-api,
```

Då kan vi også senere skru av familie-ef-soknad-api fra internett :)